src/main/java/de/doubleslash/usb_led_matrix/view/AuthenticationView.java

```diff
@@ -91,6 +91,7 @@ public void customInitialize() {
             clientIDTextField.setText(clientID);
             setTeamsIDButton.fire();
          }
+         input.close();
```

Comment:
looks weird that it is only closed when everything before succeeded. "try with" syntax would be better

Reply:
Done.